src/Shell/Task/LoadTask.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         3.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
+namespace Cake\Shell\Task;
+
+use Cake\Console\Shell;
+use Cake\Core\App;
+use Cake\Filesystem\File;
+
+/**
+ * Task for loading plugins.
+ *
+ */
+class LoadTask extends Shell
+{
+
+    public $path = null;
+    public $bootstrap = null;
+
+    /**
+     * Execution method always used for tasks
+     *
+     * @return boolean if action passed
+     *
+     */
+    public function main($plugin = null)
+    {
+
+        $this->path = current(App::path('Plugin'));
```

Comment:
This means plugins can be loaded from only one folder, making this a pretty limiting solution.